transport/tchannel/handler.go

```diff
@@ -207,80 +221,84 @@ func (h handler) callHandler(ctx context.Context, call inboundCall, responseWrit
 	}
 }
 
-type responseWriter struct {
-	failedWith         error
-	format             tchannel.Format
-	headers            transport.Headers
-	buffer             *bufferpool.Buffer
-	response           inboundCallResponse
-	isApplicationError bool
-	headerCase         headerCase
+type tchannelResponseWriter struct {
```

Comment:
Stuttering: `tchannel.tchannelResponseWriter`

Reply:
I agree and was thinking about this for a while. I felt that newResponseWriter was the best name for the interface (since it was the most generic) and this was the best name I thought of for the original responseWriter struct - any suggestions for a better name?